config/services.yml

```diff
@@ -772,44 +772,35 @@ https://cmr.uat.earthdata.nasa.gov:
         - image/tiff
       reprojection: true
 
-  - name: harmony/netcdf-to-zarr
+  # UAT GDAL
```

Comment:
The difference that git shows in UAT is a little wonky - I think that's primarily because git is picking up similarities in the structure of different service chains. All I did was move the NetCDF-to-Zarr related chains lower down (even if the diff makes it look more complicated)